src/components/activities/ActivityView/EnigmeActivityView.tsx

```diff
@@ -95,10 +96,7 @@ export const EnigmeActivityView = ({ activity, isAnswer }: EnigmeActivityViewPro
               <h3>Indice supplémentaire</h3>
               <p className="text" style={{ margin: 0, padding: 0 }}>
                 {`${enigmeType.title2} est `}
-                <strong>
-                  {(activity.data.theme === -1 ? activity.data.themeName ?? '' : enigmeData[activity.data.theme]?.step ?? '').toLowerCase()}
-                </strong>{' '}
-                !
+                <strong>{(activity.data.theme === -1 ? activity.data.themeName ?? '' : subCategories[data.theme].label ?? '').toLowerCase()}</strong>!
```

Comment:
**nit:** Should we prevent showing this clue when the enigme subtype is `-1` ?  Because in that case there's no sub-category, so no clue...